sdk/dashboard/Azure.ResourceManager.Dashboard/tests/DashboardTestEnvironment.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I believe you can remove this file, because we already have a [DashboardManagementTestEnvironment](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/dashboard/Azure.ResourceManager.Dashboard/tests/DashboardManagementTestEnvironment.cs).

Reply:
Removed thanks